### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Databases for assessment_High Contrast(Desert mode): Entire app is not adapting High contrast desert mode.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19865]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG4.3.3; #User Preferences;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Repro Steps:

1. Turn on High contrast(Desert mode)
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment" in step 1.
7. Verify whether application is adapting High contrast desert mode or not

### Actual Result:
App is not adapting High contrast desert mode.

### Expected Result:
App should adapt High contrast desert mode and all the controls should be accessible

### Attachment:
![HC_desert_](https://user-images.githubusercontent.com/83570638/176410191-c41a764c-1810-4dd2-928c-15ec3403fd87.gif)

**Comment by aasimkhan30** [comment]
@alanrenmsft , this requires us to add a new theme to ads. Is this something we want to support?

**Comment by alanrenmsft** [comment]
@aasimkhan30 similar to https://github.com/microsoft/azuredatastudio/issues/18555, we only support one high contrast theme. 

@NiharikaTipirishetty  please close the issue, this is by design.

**Comment by aasimkhan30** [comment]
Closing the issue based on previous discussion

**Comment by VijayaBharathyA** [comment]
@aasimkhan30 
Is "we only support one high contrast theme" approved by SME Could you please share details with us

**Comment by kupatkar99** [comment]
@aasimkhan30 As per the above discussion could you please share us the details which is approved by SME regarding  "we only support one high contrast theme". Thanks!

**Comment by aasimkhan30** [comment]
@alanrenmsft and @kburtram do we have plans of supporting light contrast theme in ADS? Probably by a future merge?

**Comment by aasimkhan30** [comment]
@kupatkar99 I have added the light theme to ADS. Can you please test with the latest insiders and close this issue?

**Comment by kupatkar99** [comment]
verified the issue in below environment:
Version: 1.43.0-insider (user setup)
Commit: 0741e18533b68097bf6b44953374074434c4ca53
Date: 2023-03-23T05:40:13.399Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Issue is fixed hence closing the bug. Please find added attachment for reference.
<img width="683" alt="19865_Fixed" src="https://user-images.githubusercontent.com/98530069/227197417-3c525d67-c7f1-4290-9ed1-33a4f9a85dda.png">

**Comment by NiharikaTipirishetty** [comment]
#Closed;
